### PR TITLE
python3Packages.python-pidfile: init at 3.0.0

### PR DESCRIPTION
**Pull request by legendofmiracles**
###### Motivation for this change
library needed for other program that i'd like to package

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by Kranzes — APPROVED** [review]

File: pkgs/development/python-modules/python-pidfile/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-HhCX30G8dfV0WZ/++J6LIO/xvfyRkdPtJkzC2ulUKdA=";
+  };
+
+  propagatedBuildInputs = [
+    psutil
+  ];
+
```
Comment (fabaff) [inline pkgs/development/python-modules/python-pidfile/default.nix:20]: If there are no test then `doCheck = false;` should be added. 

Let's see...https://github.com/mosquito/python-pidfile/issues/7

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 129272` at 1bbfba96 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.python-pidfile</li>
    <li>python39Packages.python-pidfile</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/no-python-tests.md">no-python-tests</a>

Test runner could not discover any test cases: ‘Ran 0 tests in 0.000s’
Near pkgs/development/python-modules/python-pidfile/default.nix:24:0:
```
   |
24 |     description = "Python context manager for managing pid files";
   | ^
```
</li>
  </ul>
</details>

File: pkgs/development/python-modules/python-pidfile/default.nix (hunk 1 above)
Comment (legendofmiracles) [reply pkgs/development/python-modules/python-pidfile/default.nix:20]: Thanks for the feedback!

File: pkgs/development/python-modules/python-pidfile/default.nix [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  psutil,
+}:
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/python-pidfile/default.nix:6]:
```suggestion
{ lib
, buildPythonPackage
, fetchPypi
, psutil
}:
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
@ofborg eval

**Comment by legendofmiracles** [comment]
Should I squash your commit @SuperSandro2000?

**Comment by SuperSandro2000** [comment]
> Should I squash your commit @SuperSandro2000?

I just wanted to wait for ofborg to be done and then squash merged them.

**Pull request merged**